src/component/chrome/chrome-container.tsx

```diff
@@ -24,13 +27,17 @@ export class ChromeContainer extends React.Component {
 		}
 
 		return (
-			<Chrome
-				leftVisible={!!previousPage}
-				rightVisible={!!nextPage}
-				onLeftClick={() => (previousPage ? store.openPage(previousPage.getId()) : undefined)}
-				onRightClick={() => (nextPage ? store.openPage(nextPage.getId()) : undefined)}
-				title={page ? page.getName() : undefined}
-			>
+			<Chrome>
+				<OverviewSwitchContainer />
+				<ViewSwitch
+					fontSize={Size.M}
+					justify="center"
+					leftVisible={!!previousPage}
```

Comment:
using `Boolean(previousPage)` would be a bit nicer

Reply:
Done ✅